src/common/substrait/src/df_logical.rs

```diff
@@ -49,10 +49,10 @@ use crate::error::{
 use crate::schema::{from_schema, to_schema};
 use crate::SubstraitPlan;
 
-pub struct DFLogicalSubstraitConvertor;
+pub struct DFLogicalSubstraitConvertorDeprecated;
```

Comment:
Why not just delete it?

Reply:
Some details are different between these two implementations. As the PR description said:

>Our implementation is kept for now for fallback. I'll remove it afterward.